source/community/index.html

```diff
@@ -27,13 +27,13 @@ <h2>How to get involved</h2>
                 <pre style="padding-left: 30px"><a href="http://bit.ly/iiif-slack">http://bit.ly/iiif-slack</a></pre></div>
 
             <div>Attend a IIIF Event:
-                <pre style="padding-left: 30px"><a href="{{ site.url }}{{ site.baseurl }}/event/">{{ site.url }}{{ site.baseurl }}/event/</a></pre></div>
+                <pre style="padding-left: 30px"><a href="/event/">{{ page.webprefix }}/event/</a></pre></div>
 
             <div>Submit a news items to the IIIF Community Newsletter:
                 <pre style="padding-left: 30px"><a href="https://goo.gl/forms/2LMe9zUHVBEbI62C3">IIIF Newsletter submission form</a></pre></div>
 
             <div>Participate in bi-weekly teleconferences:
-                <pre style="padding-left: 30px"><a href="{{ site.url }}{{ site.baseurl }}/community/call">{{ site.url }}{{ site.baseurl }}/community/call</a></pre></div>
+                <pre style="padding-left: 30px"><a href="/community/call">{{ page.webprefix }}/community/call</a></pre></div>
```

Comment:
To make internal links with api or website you need to use `{{ site.url }}{{ site.baseurl }}` rather than {{ page.webprefix }} as webprefix is https://iiif.io which means the links won't work on preview.iiif.io.